src/wave.js

```diff
@@ -322,12 +322,12 @@ export class Wave extends mix(WaveBase).with(
         this.drawBoundaries();
         if (this.isDrawBondsEnabled) this.drawBonds();
         this.render();
-        this.createLabelSpritesAsPoints();
+        this.createLabels();
         this.refillChosenAtoms();
     }
 
     render() {
-        this.adjustLabelsToCameraPosition(this.scene, this.camera);
+        if (this.settings.labelsConfig.areSpritesUsed) this.adjustLabelsToCameraPosition();
```

Comment:
Move this to create labels?

Reply:
No. Looked at it again. Adjustment should happen on render() after the camera moved, not when the labels are created.

Reply:
Let's move the if statement to the function itself